mne/decoding/ssd.py

```diff
@@ -39,35 +39,36 @@ class SSD(BaseEstimator, TransformerMixin):
         Filtering for the frequencies of non-interest.
     reg : float | str | None (default)
         Which covariance estimator to use.
-        If not None (same as 'empirical'), allow regularization for
-        covariance estimation. If float, shrinkage is used
-        (0 <= shrinkage <= 1). For str options, reg will be passed to
-        method to :func:`mne.compute_covariance`.
+        If not None (same as 'empirical'), allow regularization for covariance
+        estimation. If float, shrinkage is used (0 <= shrinkage <= 1). For str
+        options, reg will be passed to method :func:`mne.compute_covariance`.
     n_components : int | None (default None)
         The number of components to extract from the signal.
-        If n_components is None, no dimensionality reduction is applied.
+        If None, the number of components equal to the rank of the data are
+        returned (see ``rank``).
     picks : array of int | None (default None)
         The indices of good channels.
-    sort_by_spectral_ratio : bool (default False)
-       If set to True, the components are sorted accordingly
-       to the spectral ratio.
-       See Eq. (24) in :footcite:`NikulinEtAl2011`.
-    return_filtered : bool (default True)
-        If return_filtered is True, data is bandpassed and projected onto
-        the SSD components.
+    sort_by_spectral_ratio : bool (default True)
+        If set to True, the components are sorted according to the spectral
+        ratio.
+        See Eq. (24) in :footcite:`NikulinEtAl2011`.
+    return_filtered : bool (default False)
```

Comment:
reasons for changing these defaults?!

Reply:
In `__init__`, `sort_by_spectral_ratio` is default `True`, and `return_filtered` is default `False`:
https://github.com/mne-tools/mne-python/blob/4ee23004264b63ba2861ca70a4c777aee141de97/mne/decoding/ssd.py#L86-L89

however the existing documentation lists this the other way around:
https://github.com/mne-tools/mne-python/blob/4ee23004264b63ba2861ca70a4c777aee141de97/mne/decoding/ssd.py#L51-L57

so this change was just to have the documentation reflect the actual default values.